server/user.go

```diff
@@ -51,6 +51,26 @@ func (s *Server) registerUserRoutes(g *echo.Group) {
 		return nil
 	})
 
+	g.GET("/username/:id", func(c echo.Context) error {
```

Comment:
`/user/:id` might be better

Reply:
GET user by id would be more useful. But needs to remove the some sensitize fields. e.g. `OpenID`

Reply:
That's what I named it at first, but most of the user's information shouldn't be made public

Reply:
Or `/user/:id/name`?